.github/workflows/parent-pom-workaround.yml

```diff
@@ -0,0 +1,24 @@
+name: Parent POM Workaround
+on:
+  workflow_call
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: "Parent POM Workaround: Checkout pass 'main'"
+        uses: actions/checkout@v2
+        with:
+          repository: eclipse-pass/main
+      - name: "Set up JDK 14"
+        uses: actions/setup-java@v1
+        with:
+          java-version: 14
```

Comment:
Should this also be Java 8?

Reply:
In my testing, the Java version didn't matter for this piece so I left it at 14. My understanding is that all this "workaround" does is install `~/.m2/repository/org/eclipse/pass/eclipse-pass-parent/0.0.1-SNAPSHOT/eclipse-pass-parent-0.0.1-SNAPSHOT.pom`.